src/test/java/DisplayTest.java

```diff
@@ -6,12 +6,12 @@
 public class DisplayTest {
 
     private Game game;
-    private Display display;
+    private DisplayFake display;
 
     @Before
     public void setUp() {
         Board board = new Board();
-        this.game = new Game(new Player("X"), new Player("O"), board);
+        this.game = new Game(new HumanPlayer("X"), new HumanPlayer("O"), board);
```

Comment:
If as your client I said 'I want to change the symbols X and O to A and B, ideally you want to update the code in one place. At the moment the strings X and o are being duplicated across different classes and tests. Something to think about. (Same with menu options like '1' representing human and '2' representing computer.)